jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindowUI.kt

```diff
@@ -5,13 +5,21 @@ package software.aws.toolkits.jetbrains.services.sqs.toolwindow
 
 import com.intellij.openapi.project.Project
 import com.intellij.ui.components.JBTabbedPane
+import com.intellij.util.ui.JBUI
+import software.amazon.awssdk.services.sqs.SqsClient
 import software.aws.toolkits.jetbrains.services.sqs.Queue
 import software.aws.toolkits.resources.message
 
 // Will add more parameters once window is populated
-class SqsWindowUI(private val project: Project, val queue: Queue) {
+class SqsWindowUI(
+    private val project: Project,
+    private val client: SqsClient,
+    val queue: Queue
+) {
     val mainPanel = JBTabbedPane().apply {
-        this.add(message("sqs.queue.polled.messages"), PollMessagePane().component)
+        tabComponentInsets = JBUI.emptyInsets()
+        border = JBUI.Borders.empty()
+        this.add(message("sqs.queue.polled.messages"), PollMessagePane(project, client, queue).component)
```

Comment:
nit: it's weird to have `this` here but not above. With apply it's obvious what it's being applied to so `this` should be removed